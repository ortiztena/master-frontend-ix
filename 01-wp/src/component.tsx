import React from "react";
import classes from "./componentStyles.scss";


export const Component = () => {
  // 

  return (
    <div>
      <h2 className={classes.resultBackground}>
        Hello World
      </h2>
    </div>
  );
};
