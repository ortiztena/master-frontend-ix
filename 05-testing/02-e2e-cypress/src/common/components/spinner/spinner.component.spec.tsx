import * as React from 'react';
import { render, screen } from '@testing-library/react';
import { SpinnerComponent } from './spinner.component';
import * as mocked from 'react-promise-tracker/lib/trackerHook';

describe('Spinner.component spec', () => {
  it('', () => {
    //   // Arrange
    //   const promiseTrackerStub = jest
    //     .spyOn(mocked, 'usePromiseTracker')
    //     .mockImplementation(() => ({ promiseInProgress: true }));
    //   // Act
    //   render(<SpinnerComponent />);
    //   const presentation = screen.findByRole('presentation');
    //   // Assert
    //   // expect(promiseTrackerStub).toHaveBeenCalled();
    //   expect(presentation).toBeInTheDocument();
  });
});
