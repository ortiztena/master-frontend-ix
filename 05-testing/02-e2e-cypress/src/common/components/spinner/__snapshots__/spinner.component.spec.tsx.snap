// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` spinner.component spec  1`] = `<DocumentFragment />`;
